deployment/Deploy.ps1

```diff
@@ -18,6 +18,45 @@ process {
         Write-Host $Title
         Write-Host "=============================================================`n`n"
     }
+
+     # Checking Azure CLI is installed
+     $AzCliCheck= (az version | ConvertFrom-Json | Select -ExpandProperty "azure-cli")
+     if( $AzCliCheck -eq $null){
+         Write-Host "Azure CLI is not installed and please go to this link to install. (https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest%3FWT.mc_id%3Dlearnlti-github-cxa)"        
+         exit
+     }
+     else {
+         Write-Host "Azure CLI is installed."
+     }
+     # Checking .Net Framework is installed
+     $NetFrCheck= (dotnet --list-sdks)
+     if( $NetFrCheck -eq $null){
```

Comment:
Is it possible to check the right version is installed? Check their documentation, but they needed a specific version.

This applies to all the checks